pekko-sample-distributed-workers-scala/src/main/scala/worker/Main.scala

```diff
@@ -2,12 +2,12 @@ package worker
 
 import java.io.File
 import java.util.concurrent.CountDownLatch
-
 import org.apache.pekko.actor.typed.ActorSystem
-import org.apache.pekko.actor.typed.scaladsl.Behaviors
+import org.apache.pekko.actor.typed.scaladsl.{ActorContext, Behaviors}
 import org.apache.pekko.cluster.typed.Cluster
 import org.apache.pekko.persistence.cassandra.testkit.CassandraLauncher
-import com.typesafe.config.{ Config, ConfigFactory }
+import com.typesafe.config.{Config, ConfigFactory}
```

Comment:
Can you change the import back? Our scalafmt style require the spaces before/after the braces. You will need to fix the new import above too.